x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/index.tsx

```diff
@@ -135,20 +198,14 @@ const InstalledPackages: React.FC = memo(() => {
     []
   );
 
-  const categories = useMemo(
+  const categories: CategoryFacet[] = useMemo(
     () => [
       {
         id: '',
-        title: i18n.translate('xpack.fleet.epmList.allFilterLinkText', {
-          defaultMessage: 'All',
-        }),
```

Comment:
What's the reason for moving the logic for these two titles to the `CategoryFacets` component instead of leaving it here? It feels simpler to have this in one place IMO.

Reply:
discussed offline. `title` has been removed of the type. Moving it into `CategoryFacets` allows to isolate lookup of the human-readable category-name in one spot (iso. two). It also avoids having to introduce a new type with the `title` property just to define the `CategoryFacets`-interface.